spec/presenters/tree_node/ext_management_system_spec.rb

```diff
@@ -28,7 +28,6 @@
     :ems_vmware_cloud                 => { :tip_prefix => 'Ems Cloud' },
     # Other remaining providers
     :automation_manager_ansible_tower => { :key_prefix => 'at-' },
-    :configuration_manager_foreman    => { :key_prefix => 'fr-' },
```

Comment:
should the configuration_manager entry about 12 lines down be uncommented and/or deleted?

Reply:
@h-kataria Is this a valid concern? ☝️

Reply:
@Fryguy now that the tree is removed from the left side in Configuration Managers area, we do not need this entry.